pkgs/servers/http/mini-httpd/default.nix

```diff
@@ -8,6 +8,8 @@ stdenv.mkDerivation rec {
     sha256 = "1x4b6x40ymbaamqqq9p97lc0mnah4q7bza04fjs35c8agpm19zir";
   };
 
+  configureFlagsArray = [ "CFLAGS=-ansi" ];
```

Comment:
I was looking to fix this and it looks like you already did in 8e0f0776aac2c05ff02b64c7015dc91828a07e58